creduce/cvise.example.sh

```diff
@@ -21,7 +21,5 @@ fi
 cp -r "${YKLUA_HOME}/creduce/src" ./
 cd ./src
 
-make onelua
```

Comment:
Doesn't this mean you wouldn't rebuild the `onelua` binary between runs?

Reply:
ah yes, that won't work.
I need to refresh my memory on cvise